creating_reference_files/CreatingRomanFlatFieldReferenceFiles.ipynb

```diff
@@ -91,10 +93,42 @@
     "flat = rds.FlatRef()\n",
     "flat['meta'] = meta\n",
     "flat['data'] = np.ones((4096, 4096), dtype=np.float32) # You will supply a real flat, of course\n",
-    "flat['dq'] = np.zeros((4096, 4096), dtype=np.uint32) # likewise (this will soon be changed to uint16)\n",
+    "flat['dq'] = np.zeros((4096, 4096), dtype=np.uint16) # likewise (this will soon be changed to uint16)\n",
     "flat['err'] = np.zeros((4096, 4096), dtype=np.float32) # likewise"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "c63fc000",
+   "metadata": {},
+   "source": [
+    "The flat field requires a dq_def item, which is described in more detail in the associated Generic tutorial"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8fa4d66d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dq_def = np.array([\n",
```

Comment:
dq_def should be removed throughout

Reply:
Done, except as an example of how to add for INS purposes. It says it is optional.